flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -644,6 +670,16 @@ public String asSummaryString() {
 			}
 			return result;
 		}
+
+		@Override
+		public Optional<List<Map<String, String>>> listPartitions() {
+			return Optional.of(allPartitions);
+		}
+
+		@Override
+		public void applyPartitions(List<Map<String, String>> remainingPartitions) {
+			this.allPartitions = remainingPartitions;
```

Comment:
please update the `convertToRowData` method based on the values of `remainingPartitions`